package.json

```diff
@@ -12,5 +12,8 @@
     "ui",
     "lambdas/CreateItemFunction",
     "lambdas/RetrieveItemsFunction"
-  ]
+  ],
+  "devDependencies": {
+    "husky": "^4.3.8"
+  }
```

Comment:
Afaik we shouldn't have dependencies in the monorepo `package.json`, this file is just for reference to the individual workspaces themselves